include/aspect/introspection.h

```diff
@@ -205,7 +205,9 @@ namespace aspect
         unsigned int       pressure;
         unsigned int       temperature;
         std::vector<unsigned int> compositional_fields;
+        std::vector<unsigned int> compositional_field_sparsity_pattern;
       };
```

Comment:
I think it would be useful to document what that new variable actually means. This is in the `BlockIndices` structure, but I think what you are actually storing is which sparsity pattern corresponds to the various compositional fields. This is somehow *related* to the block indices, but perhaps not as obvious a match that it doesn't have to be documented.

Reply:
Correct. It is a block index (in which block do I find the matrix/sp to copy from), so I think this is the right place. Will document...